src/main/resources/hudson/plugins/jira/versionparameter/JiraVersionParameterDefinition/config.jelly

```diff
@@ -12,6 +12,9 @@
     </f:entry>
     <f:entry title="${%Show Released Versions}" field="jiraShowReleased">
```

Comment:
I think it would be good to add a help text saying that "By default when all checkboxes are unchecked, all versions are displayed" - see [inline-help](https://www.jenkins.io/doc/developer/forms/jelly-form-controls/#add-an-inline-help-to-a-form-element)